src/main/java/org/akip/domain/TaskDefinition.java

```diff
@@ -0,0 +1,292 @@
+package org.akip.domain;
+
+import org.apache.tools.ant.Task;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Table(name = "task_definition")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class TaskDefinition implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator", sequenceName = "akip_hibernate_sequence")
+    private Long id;
+
+    @Column(name = "task_id")
+    private String taskId;
+
+    @Column(name = "name")
+    private String name;
+
+    @Lob
+    @Type(type = "org.hibernate.type.TextType")
+    @Column(name = "documentation")
+    private String documentation;
+
+    @Column(name = "assignee")
+    private String assignee;
+
+    @Column(name = "candidate_users")
+    private String candidateUsers;
+
+    @Column(name = "candidate_groups")
+    private String candidateGroups;
```

Comment:
By default, the length of String fields is 255 characters. It may not be enough to store this information. Use a CLOB or a longer varchar.

Reply:
Done!